page-components/expandables.md

```diff
@@ -77,19 +80,135 @@ Standard expandables feature a colored bar, a header, the expand/collapse minico
 
 </div>
 
-<h2 id="states">States<span class="cf-code-link"><a href="https://cfpb.github.io/cf-expandables/docs/">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
+<h3 class="h4">Group</h3>
+
+<div class="content-33 content-first">
+
+Expandable sections  are used in a group, they are stacked vertically, with each row sharing its top or bottom stroke with the adjacent row. In this use case expandables have no background color of their own, but match whatever color they used on (this would typically be a white page or 5% grey well). Title text should be consistent across a group of expandables.
+
+</div>
+
+<div class="content-67 content-last">
+
+{::nomarkdown}
+<div class="expandable-group">
+    <div class="expandable-group_header">Expandable group header</div>
```

Comment:
This expandable group header pattern is outdated. Remove this line from the example.